maquetado/scss/navbar.scss

```diff
@@ -0,0 +1,23 @@
+.navbar {
+  align-items: center;
+  background-color: $nav-background-color;
+  box-shadow: 5px 5px 4px $box-shadow-color;
+  display: flex;
+  height: $nav-height;
+  justify-content: space-between;
+  position: fixed;
+  width: 100%;
+  z-index: 1;
+}
+
+.logo-img {
+  height: $nav-logo-height;
+  justify-self: flex-start;
+  margin-left: 10px;
```

Comment:
It's recommended to use padding in this cases, since the main container is the one defining the boundaries of its contents. Prefer to use margins to separate siblings instead.